contracts/MyMultiSig.cdc

```diff
@@ -28,6 +28,7 @@ pub contract MyMultiSig {
 
```

Comment:
Noticed this intent params seems not in use, if so can we clean it up in a separate PR? https://github.com/DapperCollectives/VESSEL/blob/05a324ece69f212a0884759352ad573f15bb63c3/contracts/MyMultiSig.cdc#L162

Reply:
Sure, let's create a ticket for it

Reply:
Could you please share the ticket link here? Let me know if you need help to create one.

Reply:
https://linear.app/dappercollectives/issue/VSL-109/multisignaction-cleanup